subworkflows/local/variants_ivar.nf

```diff
@@ -1,194 +1,194 @@
+
+workflow VARIANTS_IVAR {
+    take:
+    bam                 // channel: [ val(meta), [ bam ] ]
+    fasta               // channel: /path/to/genome.fasta
+    gff                 // channel: /path/to/genome.gff
+    bed                 // channel: /path/to/primers.bed
+    snpeff_db           // channel: /path/to/snpeff_db/
+    snpeff_config       // channel: /path/to/snpeff.config
+    ivar_multiqc_header // channel: /path/to/multiqc_header for ivar variants
+
+    main:
+
+    //
+    // Call variants
+    //
+    IVAR_VARIANTS ( bam, fasta, gff )
+
+    //
+    // Convert original iVar output to VCF, zip and index
+    //
+    IVAR_VARIANTS_TO_VCF ( IVAR_VARIANTS.out.tsv, ivar_multiqc_header )
+
+    VCF_BGZIP_TABIX_STATS ( IVAR_VARIANTS_TO_VCF.out.vcf )
+
+    //
+    // Create genome consensus
+    //
+    ch_consensus         = Channel.empty()
+    ch_consensus_qual    = Channel.empty()
+    ch_bases_tsv         = Channel.empty()
+    ch_bases_pdf         = Channel.empty()
+    ch_quast_results     = Channel.empty()
+    ch_quast_tsv         = Channel.empty()
+    ch_quast_version     = Channel.empty()
+    ch_pangolin_report   = Channel.empty()
+    ch_pangolin_version  = Channel.empty()
+    ch_nextclade_report  = Channel.empty()
+    ch_nextclade_version = Channel.empty()
+    if (!params.skip_consensus) {
+        IVAR_CONSENSUS ( bam, fasta )
+        ch_consensus       = IVAR_CONSENSUS.out.fasta
+        ch_consensus_qual  = IVAR_CONSENSUS.out.qual
+
+        PLOT_BASE_DENSITY ( ch_consensus )
+        ch_bases_tsv = PLOT_BASE_DENSITY.out.tsv
+        ch_bases_pdf = PLOT_BASE_DENSITY.out.pdf
+
+        if (!params.skip_variants_quast) {
+            QUAST ( ch_consensus.collect{ it[1] }, fasta, gff, true, params.gff )
+            ch_quast_results = QUAST.out.results
+            ch_quast_tsv     = QUAST.out.tsv
+            ch_quast_version = QUAST.out.version
+        }
+
+        if (!params.skip_pangolin) {
+            PANGOLIN ( ch_consensus )
+            ch_pangolin_report  = PANGOLIN.out.report
+            ch_pangolin_version = PANGOLIN.out.version
+        }
+
+        if (!params.skip_nextclade) {
+            NEXTCLADE ( ch_consensus, 'csv' )
+            ch_nextclade_report  = NEXTCLADE.out.csv
+            ch_nextclade_version = NEXTCLADE.out.version
+        }
+    }
+
+    //
+    // Annotate variants
+    //
+    ch_snpeff_vcf      = Channel.empty()
+    ch_snpeff_tbi      = Channel.empty()
+    ch_snpeff_stats    = Channel.empty()
+    ch_snpeff_csv      = Channel.empty()
+    ch_snpeff_txt      = Channel.empty()
+    ch_snpeff_html     = Channel.empty()
+    ch_snpsift_txt     = Channel.empty()
+    ch_snpeff_version  = Channel.empty()
+    ch_snpsift_version = Channel.empty()
```

Comment:
Add a line?
```suggestion
    ch_snpsift_version = Channel.empty()
         
```

Reply:
I tend to not add a line here but if I do it will mean changing stuff everywhere else too 😅

Reply:
🔕 OK, for some reason I thought that you add a line on these cases.